src/datepicker/themes/bs/bs-daterangepicker-container.component.ts

```diff
@@ -155,4 +158,10 @@ export class BsDaterangepickerContainerComponent extends BsDatepickerAbstractCom
     this._store.dispatch(this._actions.selectRange(this._rangeStack));
   }
 
+  setMaxDateRangeOnCalendar(currentSelection: Date) {
```

Comment:
Could you please add typing for returning data?

Reply:
Yes, Done